src/coreComponents/codingUtilities/StringUtilities.hpp

```diff
@@ -102,6 +102,10 @@ RETURN_TYPE tokenize( string const & str,
                       string const & delimiters,
                       bool const treatConsecutiveDelimAsOne = true );
 
+/// Subdivide string by whitespaces (see std::isspace()) (do not create any empty token).
+template< typename RETURN_TYPE = string_array >
+RETURN_TYPE tokenizeBySpaces( string const & str );
```

Comment:
I think that passing by we could update the `tokenize` function with something like
```c++
template< template< class... > class C = std::vector>
C< string > tokenize( string const & str,
                      string const & delimiters,
                      bool const treatConsecutiveDelimAsOne )
```
and use it like
```c++
std::list< string > res = tokenize< std::list >( a, b, c );
```
if you want to change the container. But the `string` will be implied.

Also, we should default to `std` containers when manipulating `string`.
They are more powerful and flexible than our HPC `LvArrays` in that precise context.
Could you make the update in the code (and use `std` containers)? There are like max 10 calls to this function.

Reply:
No problem, I will make the update right away.

Reply:
Great! First be sure to fix your initial issue. Otherwise you may end up with conflicts because the footprint of your PR is going to grow.